dizoo/box2d/car_racing/envs/car_racing_env_wrapper.py

```diff
@@ -0,0 +1,69 @@
+# Borrow a lot from openai baselines:
+# https://github.com/openai/baselines/blob/master/baselines/common/atari_wrappers.py
+
+import gym
+from collections import deque
+from ding.envs import NoopResetWrapper, MaxAndSkipWrapper, EpisodicLifeWrapper, FireResetWrapper, WarpFrameWrapper, ScaledFloatFrameWrapper, \
+                        ClipRewardWrapper, FrameStackWrapper
+
+
+def wrap_deepmind(env_id, episode_life=True, clip_rewards=True, frame_stack=4, scale=True, warp_frame=True):
```

Comment:
you can refer to this [repo](link) for more proper env wrapper for car racing. Don't need follow atari wrapper strictly.

Reply:
> you can refer to this [repo](link) for more proper env wrapper for car racing. Don't need follow atari wrapper strictly.

Sorry, I can't open the link. Could you please resend it?